game-core/src/main/java/games/strategy/triplea/delegate/UndoableMove.java

```diff
@@ -29,83 +29,84 @@
  */
 public class UndoableMove extends AbstractUndoableMove {
   private static final long serialVersionUID = 8490182214651531358L;
-  private String m_reasonCantUndo;
-  private String m_description;
+
+  private String reasonCantUndo;
+  private String description;
   // this move is dependent on these moves
   // these moves cant be undone until this one has been
-  private final Set<UndoableMove> m_iDependOn = new HashSet<>();
+  private final Set<UndoableMove> meDependOn = new HashSet<>();
```

Comment:
I realize the grammar here sucks, but `iDependOn` violates Checkstyle.  I was going to just name it `dependOn`, but then I figured the original author probably felt it was important to balance the names `iDependOn` and `dependOnMe`.

Reply:
What about `dependencies`?

Reply:
In that case, to maintain the balance with `dependOnMe`, what about the following?

* `iDependOn` -> `dependencies`
* `dependOnMe` -> `dependents`

Reply:
👍

Reply:
ed7e1413f